src/style.css

```
@@ -0,0 +1,106 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+.search-image-container {
+  position: relative;
+}
+
+.search-image-container::before {
+  content: "";
+  position: absolute;
+  background: url(images/Search.svg) no-repeat;
+  top: 25px;
+  bottom: 0;
+  left: 26px;
+  margin: auto 0;
+  width: 25px;
+}
+
+.search-bar-big {
+  margin-top: 20px;
+}
+
+.search-field-big {
+  height: 70px;
+  width: 100%;
+  border-radius: 4px;
+  padding-left: 62px ;
+  border: 1px solid #e1e7ed;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.search-field-big:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-field-big:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.search-bar-small {
+  margin-top: 20px;
+}
+
+.search-image-container-small {
+  position: relative;
+}
+   
+.search-image-container-small::before {
+  content: "";
+  position: absolute;
+  background: url(images/Search.svg) no-repeat;
+  left: 13px;
+  top: 15px;
+  bottom: 16px;
+  margin: auto 0;
+  width: 11px;
+  height: 11px;
+  background-size: 11px;
+}
+
+.search-field-small {
+  height: 42px;
+  width: 100%;
+  border-radius: 4px;
+  padding-left: 33px;
+  border: 1px solid #e1e7ed;
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 22px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.search-field-small:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-field-small:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.search-field-big::placeholder {
+  font-size: 16px;
+  color: #3d4e61;
+  font-weight: 400;
+  line-height: 21.86px;
```

Do not use fractional numbers
```suggestion
  line-height: 22;
```